chemprop/cli/hpopt.py

```diff
@@ -374,15 +435,32 @@ def tune_model(
         tune_config=tune_config,
     )
 
-    return tuner.fit()
+    try:
+        return tuner.fit()
+    except OSError as e:
+        if "AF_UNIX path length cannot exceed 107 bytes" in str(e):
+            raise OSError(
+                f"Ray Tune fails due to: {e}. This can sometimes be solved by providing a temporary directory, num_cpus, and num_gpus to Ray Tune via the CLI: --raytune-temp-dir <absolute_path> --raytune-num-cpus <int> --raytune-num-gpus <int>."
+            )
+        else:
+            raise e
 
 
 def main(args: Namespace):
     if NO_RAY:
         raise ImportError(
-            "Ray Tune requires ray to be installed. Use 'pip install -U ray[tune]' to install."
+            "Ray Tune requires ray to be installed. If you installed Chemprop from PyPI, make sure that your Python version is 3.11 and use 'pip install -U ray[tune]' to install ray. If you installed from source, use 'pip install -e .[hpopt]' in Chemprop folder to install all hpopt relevant packages."
         )
 
+    if not ray.is_initialized():
+        ray.init(
+            _temp_dir=args.raytune_temp_dir,
+            num_cpus=args.raytune_num_cpus,
+            num_gpus=args.raytune_num_gpus,
+        )
```

Comment:
```suggestion
        try:
            ray.init(
                _temp_dir=args.raytune_temp_dir,
                num_cpus=args.raytune_num_cpus,
                num_gpus=args.raytune_num_gpus,
            )
        except OSError as e:
            if "AF_UNIX path length cannot exceed 107 bytes" in str(e):
                raise OSError(
                    f"Ray Tune fails due to: {e}. This can sometimes be solved by providing a temporary directory, num_cpus, and num_gpus to Ray Tune via the CLI: --raytune-temp-dir <absolute_path> --raytune-num-cpus <int> --raytune-num-gpus <int>."
                )
            else:
                raise e
```